### PR TITLE
Rename Download -> Save.

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**
- We want ‘Send to micro:bit’ to be the preferred route for flashing -
  if you can’t use WebUSB it downloads and gives you instructions anyway
  and if it fails then the download route is now a clear alternative

- We know that some people aren’t sure how to save to backup their work
  and don’t realise that download does the same thing. Save is the label
  more folk are familiar with.

- Long term if we have other ways to save (say cloud storage) then Save
  is the better label to hang these off.

### CODE REVIEW

**Comment by microbit-matt-hillsdon** [comment]
This PR requires an upgrade of the theme package before merge.

File: src/e2e/app.ts
Comment (microbit-robert) [inline src/e2e/app.ts:799]:
```suggestion
   * Trigger a hex file save and wait for the download to complete.
```

Comment (microbit-matt-hillsdon) [reply src/e2e/app.ts:799]: `ifle` feels like it could be a Python keyword 😄

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Comment by microbit-matt-hillsdon** [comment]
> This PR requires an upgrade of the theme package before merge.

Done (and rebased).

**Comment by microbit-matt-hillsdon** [comment]
Will merge when CI passes.

**Pull request merged**
